### PR TITLE
Batch fixes for issues

### PR DESCRIPTION
**Pull request by JackLivio**
Batched issues and their corresponding pr fixes:

- #134 fixed by #157 
- #142 fixed by #156 
- #151 fixed by #155 
- #145 fixed indirectly by #155 
- #146 fixed by #154 
- #67 fixed by #158

### CODE REVIEW

**Comment by jacobkeeler** [comment]
The fix for #134 doesn't seem to work, the timer text is still black in the dark theme

**Comment by jacobkeeler** [comment]
Regarding #151, Core still sends `remoteControl:true` and `videoStreaming:true` in `hmiCapabilities` because these are not included in the response from the generic HMI

**Comment by JackLivio** [comment]
@jacobkeeler 

> Regarding #151, Core still sends remoteControl:true and videoStreaming:true in hmiCapabilities because these are not included in the response from the generic HMI

Couple of problems here.

The generic HMI does not "know" about these values (unsupported) and core is setting those capabilities to true because of the contents in the hmi_capabilities.json.

`remoteControl` is missing from the HMICapabilities struct in the HMI_API. 

The RC.GetCapabilitiesResponse could set the rc_supported field but that would require the HMI to respond to RC.GetCapabilities.

File: src/js/containers/MediaTrackInfo_c.js [hunk 1]
```diff
@@ -14,11 +14,36 @@ const mapStateToProps = (state) => {
     }
     var startDate = startTime ? new Date(0, 0, 0, startTime.hours, startTime.minutes, startTime.seconds, 0) : null
     var endDate = new Date(0, 0, 0, endTime.hours, endTime.minutes, endTime.seconds, 0)
+
+    //Assign color scheme to props
+    var theme = state.theme
+    var colorScheme = {};
+    if (theme === true) { //Dark theme
+        if(app.nightColorScheme) {
+            if(app.nightColorScheme.secondaryColor) {
```
[the remark this answers is not in the document]
Comment (JackLivio) [reply src/js/containers/MediaTrackInfo_c.js]: Fixed

File: src/js/Controllers/DisplayCapabilities.js [hunk 2]
```diff
@@ -83,6 +83,14 @@ let capabilities = {
 				}
 			],
 			"imageFields": [{
+          "name": "choiceImage",
```
[the remark this answers is not in the document]
Comment (JackLivio) [reply src/js/Controllers/DisplayCapabilities.js]: Converted all spaces to tabs

File: src/js/reducers.js [hunk 3]
```diff
@@ -127,28 +127,31 @@ function ui(state = {}, action) {
...
-                    cmdID: cmdID,
-                    parentID: menuParams.parentID,
-                    position: menuParams.position,
-                    menuName: menuParams.menuName,
-                    cmdIcon: cmdIcon
-                })
+                // Push item to front of list if position is 0
+                (menuParams.position === 0) ? 
+                    subMenu.subMenu.unshift(menuItem) : 
+                    subMenu.subMenu.push(menuItem);
```
[the remark this answers is not in the document]
Comment (JackLivio) [reply src/js/reducers.js]: Fixed

**Comment by jacobkeeler** [comment]
@JackLivio seems that this is an issue with Core. It should send false for `videoStreaming` if that `Navigation` component is not available and false for `remoteControl` if the `RC` component is not available.

**Pull request merged**